sphinx_gallery/tests/test_notebook.py

```diff
@@ -158,11 +161,88 @@ def test_headings():
     assert "# White space above\n" in text
 
 
+def test_notebook_images(gallery_conf):
```

Comment:
Would it be manageable to use pytest parameterize for the first 4 tests and put the last 2 'tests' (the data URI and the missing file) in a separate test? I think it would make this easier to maintain.

Reply:
Done